README.md

```diff
@@ -1,122 +1,58 @@
 # Workshops
 
+This is the home base of ClojureBridge workshops.
+
+---
+
+## Contents
+
+### [:rocket: Getting Started](#rocket-getting-started)
+### [:memo: Guide to Listing Your Event on the Website](./docs/workshop-listing-guide.md)
+### [:books: Helpful Resources](#books-helpful-resources)
+### [:woman_technologist: Contributing](#woman_technologist-contributing)
+
+---
+
+## :rocket: Getting Started
+
 This is the repository of ClojureBridge workshops. Please file an issue to
 get started. Board members are watching this repository. If an organizer needs
 help, add a comment to the issue.
 
-Every workshop should have a mentor from one of current (or past) board members.
-Even if you have experience organizing a ClojureBridge workshop in the past, you 
-should contact your mentor. This is because things are often updated.
+When you open an issue, you will be assigned a mentor from the ClojureBridge board.
```

Comment:
```suggestion
When you open an issue, you will be assigned a mentor from the [ClojureBridge board](https://github.com/ClojureBridge/board).
```

When I first read through these docs, I was always curious to know who was actually on the board. This link might be helpful to others 😄